float64 s
float64 s_dot
float64 s_ddot
float64 d
float64 d_dot
float64 d_ddot
float64 yaw_s